string time
uint16 orderNumber
string item